marlowe-playground-client/src/Simulation.purs

```diff
@@ -144,11 +145,11 @@ handleAction _ StartSimulation = do
   moveToSlot zero
 
 handleAction _ (MoveSlot slot) = do
-  maybeExecutionState <- use (_currentMarloweState <<< _executionState)
+  maybeInTheFuture <- peruse (_currentMarloweState <<< _executionState <<< _Just <<< _slot <<< to ((>) slot))
```

Comment:
better to use https://pursuit.purescript.org/packages/purescript-prelude/4.1.1/docs/Data.Ord#v:greaterThan I think

Reply:
as discussed, you could make a new function in `Data.Lens.Extra` that is like `peruse` but for `is`, then you could have
```purescript
inTheFuture <- stateIs (_currentMarloweState <<< _executionState <<< _Just <<< _slot <<< nearly (greaterThan slot))
```
See https://pursuit.purescript.org/packages/purescript-profunctor-lenses/6.2.0/docs/Data.Lens.Prism#v:nearly for some help